### PR TITLE
Translation from martinascholger/TEI

### PR DESCRIPTION
**Pull request by martinascholger**

### CODE REVIEW

File: P5/Source/Specs/acquisition.xml [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
 See the file COPYING.txt for details
 $Date$
 $Id$
--->
-<?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
+-->
+<?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
 <elementSpec xmlns="http://www.tei-c.org/ns/1.0" module="msdescription" xml:id="ACQUISITION" ident="acquisition">
   <gloss xml:lang="en" versionDate="2007-06-12">acquisition</gloss>
+  <gloss xml:lang="de" versiondate="2021-10-24">enthält Informationen über den Vorgang, durch den ein Manuskript, ein Manuskriptteil oder ein anderes Objekt in den Besitz einer Institution gelangt ist.</gloss>
```
Comment (sabineseifert) [inline P5/Source/Specs/acquisition.xml:12]: That should be `<desc>` instead of `<gloss>`?!

Comment (sabineseifert) [inline P5/Source/Specs/acquisition.xml:12]: It also should be `@versionDate` with an upper case 'D'.

File: P5/Source/Specs/actor.xml [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
 Dual-licensed under CC-by and BSD2 licences 
 See the file COPYING.txt for details
 $Date$
 $Id$
---><?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?><elementSpec xmlns="http://www.tei-c.org/ns/1.0" module="drama" ident="actor">
+-->
+<?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
+<elementSpec xmlns="http://www.tei-c.org/ns/1.0" module="drama" ident="actor">
   <desc versionDate="2012-12-27" xml:lang="en">contains the name of an actor appearing within a cast list.</desc>
+  <desc versiondate="2021-10-25" xml:lang="de">enthält den Namen einer Schauspielerin oder eines Schauspielers, der in einer Besetzungsliste aufscheint.</desc>
```
Comment (sabineseifert) [inline P5/Source/Specs/actor.xml:12]: It should be `@versionDate` with an upper case 'D'.

File: P5/Source/Specs/annotation.xml [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
--->
-<?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
-<elementSpec xmlns="http://www.tei-c.org/ns/1.0" ident="annotation" mode="add" module="linking"
-  xmlns:sch="http://purl.oclc.org/dsdl/schematron">
+-->
+<?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
+<elementSpec xmlns="http://www.tei-c.org/ns/1.0" ident="annotation" mode="add" module="linking" xmlns:sch="http://purl.oclc.org/dsdl/schematron">
   <desc versionDate="2020-07-31" xml:lang="en">represents an annotation following the <ref target="#WADM">Web
       Annotation Data Model</ref>.</desc>
+  <desc versiondate="2021-10-21" xml:lang="de" versionDate="2021-10-21">repräsentiert eine Annotation die dem <ref target="#WADM">Web Annotation Data Model</ref>.</desc>
```
Comment (sabineseifert) [inline P5/Source/Specs/annotation.xml:13]: There are two `@versionDate` attributes: one with upper case 'D', the other with lower case 'd'.

Comment (sabineseifert) [inline P5/Source/Specs/annotation.xml:13]: There is a comma missing after "Annotation" and there is also a verb missing at the end of the sentence, something like "folgt".

**Review by sabineseifert — CHANGES_REQUESTED** [review]
Thanks for the pull requests. See individual comments for requested changes.

**Comment by martinascholger** [comment]
I think this was an early test with the translation tool. I think we should decline this PR and create a new one.

**Comment by sabineseifert** [comment]
Closing since this was a test for the translation tool.

**Pull request closed without merging**